jadx-cli/src/main/java/jadx/cli/JadxCLIArgs.java

```diff
@@ -254,4 +269,59 @@ public boolean isRespectBytecodeAccessModifiers() {
 	public boolean isExportAsGradleProject() {
 		return exportAsGradleProject;
 	}
+
+	public boolean isRenameCaseSensitive() {
+		return renameFlags.contains(RENAME.CASE);
+	}
+
+	public void setRenameCaseSensitive(boolean renameCase) {
+		if (renameCase && !isRenameCaseSensitive()) {
+			renameFlags.add(RENAME.CASE);
+		} else if (!renameCase && isRenameCaseSensitive()) {
+			renameFlags.remove(RENAME.CASE);
+		}
+	}
+
+	public boolean isRenameValid() {
+		return renameFlags.contains(RENAME.VALID);
+	}
+
+	public void setRenameValid(boolean renameValid) {
+		if (renameValid && !isRenameValid()) {
+			renameFlags.add(RENAME.VALID);
+		} else if (!renameValid && isRenameValid()) {
+			renameFlags.remove(RENAME.VALID);
+		}
+	}
+
+	public boolean isRenamePrintable() {
+		return renameFlags.contains(RENAME.PRINTABLE);
+	}
+
+	public void setRenamePrintable(boolean renamePrintable) {
+		if (renamePrintable && !isRenamePrintable()) {
+			renameFlags.add(RENAME.PRINTABLE);
+		} else if (!renamePrintable && isRenamePrintable()) {
+			renameFlags.remove(RENAME.PRINTABLE);
+		}
+	}
+
+	static class RenameConverter implements IStringConverter<Set<RENAME>> {
+
+		@Override
+		public Set<RENAME> convert(String value) {
+			value = value.toUpperCase(Locale.ROOT);
+			Set<RENAME> set = new HashSet<>();
+			if (value.equals("ALL")) {
+				set.add(RENAME.CASE);
+				set.add(RENAME.VALID);
+				set.add(RENAME.PRINTABLE);
+			} else if (!value.equals("NONE")) {
+				for (String s : value.split(",")) {
+					set.add(RENAME.valueOf(s));
```

Comment:
Throw a detailed exception like `<value> is unknown, possible values list: <list of values>`, because user can input incorrect names.
Also, add some tests to check this converter.